packages/back-end/src/controllers/experiments.ts

```diff
@@ -477,6 +481,7 @@ export async function postExperiments(
   >
 ) {
   const { org, userId } = getOrgFromReq(req);
+  const organization = await getOrganizationById(org.id);
```

Comment:
Don't need this. Org above is already the Organization object

Reply:
whoops!